charts/haproxy/v1.4/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 HAProxy Technologies LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+name: haproxy
+version: 1.1.1
```

Comment:
The parent folder v1.4/ has to be renamed to v1.1.1/
Is named after the chart version indicated here rather than the version of the app it wraps.

Reply:
> The parent folder v1.4/ has to be renamed to v1.1.1/
> Is named after the chart version indicated here rather than the version of the app it wraps.

Ah, thank you for this info. I have updated.